tests/test_suite.py

```diff
@@ -27,7 +27,7 @@ def test_suite(device):
                       for i in range(suite.component_list.count())]
     assert len(child_displays) == len(device._sub_devices)
     # Default tools are loaded
-    assert len(suite.tools) == 2
+    assert len(suite.tools) == 3
```

Comment:
:tada:

Reply:
I don't like this test. Wasn't sure where to place a `default_tools` list though...